src/ert/job_queue/queue.py

```diff
@@ -49,6 +49,11 @@
 logger = logging.getLogger(__name__)
 
 LONG_RUNNING_FACTOR = 1.25
+"""If STOP_LONG_RUNNING is true, realizations taking more time than the average
```

Comment:
Great addition!

Reply:
Just realized that these attribute docstrings are not standardized, and cannot be picked up by any tool.